File: proy/imperativo/programas/subrutina.py
def calcular_area(base, altura):
  area = (base * altura) / 2
  return area

base = float(input("Ingrese la base del triángulo: "))
altura = float(input("Ingrese la altura del triángulo: "))
area = calcular_area(base, altura)
print("El área del triángulo es:", area)

File: proy/imperativo/programas/subrutina.js
function calcularArea(base, altura) {
  let area = (base * altura) / 2;
  return area;
}

let base = parseFloat(prompt("Ingrese la base del triángulo: "));
let altura = parseFloat(prompt("Ingrese la altura del triángulo: "));
let area = calcularArea(base, altura);
alert("El área del triángulo es: " + area);